lib/create-contentful-api.ts

```diff
@@ -96,6 +97,15 @@ export default function createContentfulApi<OptionType extends ChainOptions>(
     return baseUrl
   }
 
+  const alphaFeatures = (http.httpClientParams as any as CreateClientParams)?.alphaFeatures
+
+  function maybeEnableSourceMaps(query: Record<string, any> = {}): Record<string, any> {
+    if (alphaFeatures?.withContentSourceMaps) {
+      query.includeContentSourceMaps = true
```

Comment:
We should validate against enabling Content Source Maps for CDA and report a user friendly error.

Reply:
could the `alphaFeatures?.withContentSourceMaps` part be done in the `checkIncludeContentSourceMapsParamIsAllowed` function? Then you could simply check isAllowed

nit: `areAllowed`

Reply:
sounds good 👌🏽i had picked `isAllowed` since we are checking if the param is allowed not necessarily if content source maps are allowed but it's fine either way I'll push the change

Reply:
Ah right makes sense then sorry, yeah also happy with either